jest.config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "collectCoverage": false,
+  "collectCoverageFrom": ["src/**/*.ts"],
+  "setupFilesAfterEnv": ["./src/test/suite-setup.ts"],
+  "coveragePathIgnorePatterns": [
+    "node_modules",
+    "<rootDir>/src/.*/__tests__/",
+    "<rootDir>/src/loaders",
+    "<rootDir>/src/routes",
+    "<rootDir>/src/server/*",
+    "<rootDir>/dist"
+  ],
+  "coverageDirectory": "<rootDir>/coverage/",
+  "coverageThreshold": {
```

Comment:
How did you decide on these numbers?

Reply:
everything should be at least 50 in the beginning, it was just a coverage test